docs/getting-started/README.md

```diff
@@ -166,7 +166,7 @@ minimum.
 - GitHubOrg: The GitHub org you are using for this getting-started.
 - GitHubUser: Your GitHub username.
 - GitHubRepo: The repo we will be using for this example.
-- ExternalDomain: Update this to be the to something other then `demo.iancoffey.com`
+- GitHubDomain: Update this to be the to something other then `demo.iancoffey.com`
```

Comment:
hmm it seems like these are both important to the way the task works, but do different jobs - ExternalDomain is required, so im not sure it makes sense to completely remove it from these docs (https://github.com/tektoncd/triggers/blob/a545f76e7f0021ee13a1c22b6703dae18d2639fa/docs/getting-started/create-webhook.yaml)

if anything we probably want to add a bullet for GitHubDomain - explaining that you may need to override it if you're using github enterprise